lang/guile2/Portfile

```diff
@@ -0,0 +1,172 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           xcode_workaround 1.0
+
+name                guile2
+categories          lang
+maintainers         nomaintainer
+license             LGPL-2.1+
+
+version             2.2.7
+revision            0
+checksums           rmd160  8c3dba807c4d236da81373e593d61007812ae236 \
+                    sha256  44b4c5fbbe257ccdebea18420212c9b3e90c3c86a54920d8554039fc6769a007 \
+                    size    18129833
+
+compiler.c_standard 2011
+
+# https://trac.macports.org/ticket/59365
+xcode_workaround.fixed_xcode_version 11.2
+xcode_workaround.type avoid_xcode_compiler
+
+platform darwin powerpc {
+    post-extract {
+        # the prebuilt guile binaries are broken for 32-bit darwin PPC
+        move ${worksrcpath}/prebuilt/32-bit-big-endian ${worksrcpath}/prebuilt/32-bit-big-endian-disabled
+
+        ui_msg "*** This build can take some time, as guile has to do a full bootstrap on PPC."
+    }
+}
+
+# Failed to destroot guile:
+# /opt/local/lib/guile/2.0/ccache/ice-9/and-let-star.go differs in ... and
+# cannot be merged
+universal_variant   no
+
+description         GNU's Ubiquitous Intelligent Language for Extension (guile)
+long_description    \
+        Guile is an interpreter for the Scheme programming \
+        language, packaged for use in a wide variety of \
+        environments. Guile implements Scheme as described in \
+        the Revised^5 Report on the Algorithmic Language Scheme \
+        (usually known as R5RS), providing clean and general \
+        data and control structures. \
+\
+        Guile goes beyond the rather austere language presented \
+        in R5RS, extending it with a module system, full access \
+        to POSIX system calls, networking support, multiple \
+        threads, dynamic linking, a foreign function call \
+        interface, powerful string processing, and many other \
+        features needed for programming in the real world.
+
+distname            guile-${version}
+homepage            https://www.gnu.org/software/guile/guile.html
+master_sites        gnu:guile
+
+depends_lib         port:readline \
+                    port:gettext \
+                    port:libiconv \
+                    port:libtool \
+                    port:gmp \
+                    port:libunistring \
+                    port:boehmgc \
+                    port:libffi \
+                    port:ncurses
```

Comment:
While this may seem nit-picky, it's preferable to alphabetize the various dependencies. Makes it easier to see what's what, particularly when the dependency list is lengthy.